common/pkg/server/server.go

```diff
@@ -94,9 +94,9 @@ func (s *Server) Start(ctx context.Context) error {
 		}
 
 		if t, ok := h.(lifecycle.AfterControlPlaneInitialized); ok {
-			if err := webhookServer.AddExtensionHandler(server.ExtensionHandler{
+			if err := webhookServer.AddExtensionHandler(runtimeserver.ExtensionHandler{
 				Hook:        runtimehooksv1.AfterControlPlaneInitialized,
-				Name:        strings.ToLower(h.Name()),
+				Name:        strings.ToLower(h.Name()) + "aftercontrolplaneinitialized",
```

Comment:
I wonder if we can shorten this suffix? And add a `-` separator
```suggestion
				Name:        strings.ToLower(h.Name()) + "-acpi",
```

Reply:
sounds great to me. i just did this to see if this would solve the issue that i was running into.